backend/src/main/java/com/woowacourse/levellog/levellog/application/LevellogService.java

```diff
@@ -43,6 +47,17 @@ public LevellogDto findById(final Long levellogId) {
         return LevellogDto.from(levellog);
     }
 
+    public LevellogsDto findAllByAuthorId(final Long authorId) {
+        final Member author = getMember(authorId);
+
+        final List<Levellog> levellogs = levellogRepository.findAllByAuthor(author);
```

Comment:
```suggestion
        final List<Levellog> levellogs = levellogRepository.findAllByAuthorId(authorId);
```
어차피 LevellogWithIdDto를 생성할 때 멤버 객체를 가져 오겠지만 코드 상 더 깔끔해 보이는 거 같아요👀

Reply:
사실 존재하지 않는 member 요청에 대한 검증을 하기 위해서 해당 `getMember()`를 사용하도록 코드를 작성했습니다.